assets/js/protocol/protocol-footer.js

```diff
@@ -14,22 +23,19 @@
         if (typeof Mzp.Supports !== 'undefined' && typeof Mzp.Details !== 'undefined') {
 
             // check browser supports matchMedia
-            if(Mzp.Supports.matchMedia) {
+            if (Mzp.Supports.matchMedia) {
                 var _mqWide = matchMedia('(max-width: 479px)');
 
                 // initialize details if screen is small
                 if (_mqWide.matches) {
                     Mzp.Details.init(footerHeadings);
                 }
 
-                // remove details if screen is big
-                _mqWide.addListener(function(mq) {
-                    if (mq.matches) {
-                        Mzp.Details.init(footerHeadings);
-                    } else {
-                        Mzp.Details.destroy(footerHeadings);
-                    }
-                });
+                if (window.matchMedia('all').addEventListener) {
+                    _mqWide.addEventListener('change', screenChange);
+                } else if (window.matchMedia('all').addListener) {
```

Comment:
Doesn't look like we may need this fallback, since `addEventListener` has more support than `addListener`. Going to discuss with team

Reply:
The fallback is needed for IE I believe

Reply:
**Note**: Turns out I missed looking up the `matchMedia` method during research. Since that method returns a `MediaQueryList` object, it should be noted that `MediaQueryList.addEventListener` doesn't work in Safari and IE, as mentioned here under Known Issues: https://caniuse.com/?search=matchMedia, so a fallback will be necessary as Alex pointed out above.